src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskSearchControllerCFTTest.java

```diff
@@ -740,7 +740,7 @@ public void should_have_consistent_unassigned_state_in_camunda_and_cft_db() {
 
         camundaTask.then().assertThat()
             .statusCode(HttpStatus.OK.value())
-            .body("taskState.value", is("unassigned"))
+            .body("taskState.value", is("unconfigured"))
```

Comment:
tests fail here due to value = unassigned

Reply:
Corrected.